core/internal/tensorboard/emitter.go

```diff
@@ -300,6 +297,48 @@ func (e *tfEmitter) EmitImage(
 	return nil
 }
 
+func (e *tfEmitter) verifyAndGetImagesMetadata(images []wbvalue.Image) (format string, height int, width int, err error) {
+	// Tensorboard encodes all images to png format.
+	// https://github.com/tensorflow/tensorboard/blob/b56c65521cbccf3097414cbd7e30e55902e08cab/tensorboard/plugins/image/summary.py#L85
+	format = images[0].Format
+	// Tensorboard gets the image dimensions from the first two values in the tensor.
+	//https://github.com/tensorflow/tensorboard/blob/b56c65521cbccf3097414cbd7e30e55902e08cab/tensorboard/plugins/image/summary.py#L93-L94
```

Comment:
Similarly, I was confused about the relevance here before I remembered the context :) how about

> All images in a TensorBoard image summary have the same width and height.